experimenter/pyproject.toml

```diff
@@ -64,7 +64,9 @@ djangorestframework-dataclasses = "^1.3.0"
 graphene-django = "^3.2.0"
 pyright = "^1.1.291"
 django-types = "^0.19.1"
+mozilla-nimbus-shared = "^2.5.2"
```

Comment:
Beth just removed this dependency in the [latest commit](https://github.com/mozilla/experimenter/commit/ad9134c36fdbadefd92a9a4f1b5493c967f60fa2#diff-86347d9aaa24ca8462da5352e18d80569f750e036764c45a5d3ef6e7c0fa33d3), so I'm guessing this was accidentally added back in resolving a conflict? In any case, we definitely shouldn't be adding it back in here.